certified-connectors/Rainbird/apiDefinition.swagger.json

```diff
@@ -0,0 +1,865 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Rainbird is a no-code intelligent automation platform that enables you to automate complex decisions. This connector enables you to interact with knowledge maps you\u2019ve built in Rainbird.",
+    "contact": {
+      "name": "Rainbird Technologies Ltd",
+      "url": "https://rainbird.ai/contact-us/",
+      "email": "support@rainbird.zohodesk.eu"
+    },
+    "title": "Rainbird",
+    "version": "1.2.0"
+  },
+  "host": "enterprise-api.rainbird.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/start/{kmID}": {
+      "get": {
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "environment",
+            "type": "string",
+            "enum": [
+              "enterprise-api.rainbird.ai",
+              "api.rainbird.ai"
+            ],
+            "required": true,
+            "description": "Account environment: Community (api.rainbird.ai) or Enterprise (enterprise-api.rainbird.ai).",
+            "x-ms-summary": "Environment",
+            "x-ms-visibility": "important"
+          },
+          {
+            "description": "The knowledge map ID to start a session against",
+            "in": "path",
+            "name": "kmID",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Knowledge Map ID (kmId)",
+            "x-ms-url-encoding": "single",
+            "x-ms-visibility": "important"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Session start information",
+            "schema": {
+              "example": {
```

Comment:
@Mat-Rig can you please remove example schemas from all operations?

Reply:
Hello, I am not sure how to treat that one. Should I leave it blank or write something in the tags anyways?
I corrected this by putting `string`. This is the only occurrence of such secret-look-a-like value.

Reply:
Hi @Mat-Rig! The string is OK here, thank you very much for fixing it!